content/docs/14-troubleshooting/30-pcg.md

```diff
@@ -121,4 +124,135 @@ While these are the most common errors, some other issues might be related to th
 
 
 4. If the problem persists, contact our support team by sending an email to [support@spectrocloud.com](mailto:support@spectrocloud.com).
-<br />
\ No newline at end of file
+<br />
+
+
+# Scenario - IP Address Not Assigned During PCG Creation
+
+When deploying a PCG in VMware vCenter, the IP address fails to get assigned to the new node. This can be caused by networking errors such as DHCP issues or improper execution of cloud-init.
```

Comment:
```suggestion
When deploying a PCG in VMware vCenter, the IP address fails to get assigned to the new node. This can be caused by networking errors such as DHCP issues or an incomplete cloud-init.
```
Execute is an aggressive term, so we try to avoid it.